sentences/es/cover_HassOpenCover.yaml

```diff
@@ -5,14 +5,13 @@ intents:
       - sentences:
           - "<abre> <name>"
           - "<abre> <name> <area>"
-      - sentences:
-          - "<abre> <puerta> [(del|de|de la)] (garaje|cochera)"
-        slots:
-          device_class: "garage_door"
       - sentences:
           - "<abre> <puerta> [<area>]"
         slots:
-          device_class: "gate"
+          device_class:
+            - "gate"
+            - "garage_door"
```

Comment:
Device class for garage is "garage" (https://www.home-assistant.io/integrations/cover/)